src/asn1_types/integer.rs

```diff
@@ -276,7 +276,8 @@ impl_int!(u128 => i128);
 ///
 /// Encoding an `Integer` to DER
 ///
-/// ```
+#[cfg_attr(feature = "std", doc = r#"```"#)]
+#[cfg_attr(not(feature = "std"), doc = r#"```rust,compile_fail"#)]
```

Comment:
Oh, that's an interesting trick to handle the compilation failure. Maybe there is another solution: put the doc  code in a function, and mark the function with
`/// # #cfg(feature = "std")`
This would still be compiled for the docs.rs
I have not checked, just a guess.

Note: that's just a comment, no action needed